src/main/java/com/example/bookstoreapp/models/CatalogItem.java

```diff
@@ -10,20 +10,23 @@
 @NoArgsConstructor
 public class CatalogItem {
 
-  private Long id;
```

Comment:
This class also re-written which is not required, just for itemLimit field.

This happens when you select all and do formatting. You should format only the code which is newly added.

Reply:
ok I understod what is causing these seem as newly added code ,I will keep this in consideration to avoid post this.